src/api-service/__app__/onefuzzlib/azure/image.py

```diff
@@ -21,12 +21,20 @@ def get_os(region: Region, image: str) -> Union[Error, OS]:
     client = get_compute_client()
     parsed = parse_resource_id(image)
```

Comment:
Not a problem with your PR, but let's document the caller's burden / lack of guarantees:
```suggestion
    # The dict returned here may not have any defined keys.
    # 
    # See: https://github.com/Azure/msrestazure-for-python/blob/v0.6.3/msrestazure/tools.py#L134
    parsed = parse_resource_id(image)
```